docs/guides/Uniq Factory Management/factory-purchase-options-examples.md

```diff
@@ -0,0 +1,330 @@
+    "memo": ""
+  }
+]' -p factory.manager
+```
+:::
+
+When purchasing no extra input is required from the user, the verification of groups membership will be verified by the smart contract automatically
+
+::: details purchase.a
+```sh
+cleos push action eosio.nft.ft purchase.a '[
+  {
+    "token_factory_id": 100,
+    "index": 0,
+    "max_price": "100.00000000 UOS",
+    "buyer": "alice",
+    "receiver": "alice",
+    "promoter_id": null,
+    "user_uniqs": null,
+    "memo": ""
+  }
+]' -p alice
+```
+:::
+
+## Using purchase option for swapping
+
+"Swapping" in this case implies the process where user looses ownership of his Uniq, the Uniq gets destroyed in the process and user gets a new Uniq from the factory instead. Example below requires user to give up two Uniqs: one from factory 43 and one from factory 44. Note how `strategy` is set to 1
+
+::: details setprchsreq.a
+```sh
+cleos push action eosio.nft.ft setprchsreq.a '[
+  {
+    "token_factory_id": 100,
+    "index": 0,
+    "price": "50.00000000 UOS",
+    "purchase_limit": null,
+    "promoter_basis_point": 100,
+    "purchase_option_with_uniqs": {
+        "transfer_tokens_receiver_account": null,
+        "factories": [{
+            "token_factory_id": 43,
+            "count": 1,
+            "strategy": 1
+        },{
+            "token_factory_id": 44,
+            "count": 1,
+            "strategy": 1
+        }]
+    },
+    "sale_shares": [],
+    "maximum_uos_payment": null,
+    "group_restriction": null,
+    "purchase_window_start": null,
+    "purchase_window_end": null,
+    "memo": ""
+  }
+]' -p factory.manager
+```
+:::
+
+Purchasing from such a purchase option requires user to specify which Uniqs he is willing be give up. Here assume token 123 is from factory 43 and token 124 is from factory 44. Note how the `strategy` matches the value of the purchase option
+
+::: details purchase.a
+```sh
+cleos push action eosio.nft.ft purchase.a '[
+  {
+    "token_factory_id": 100,
+    "index": 0,
+    "max_price": "100.00000000 UOS",
+    "buyer": "alice",
+    "receiver": "alice",
+    "promoter_id": null,
+    "user_uniqs": {
+      "tokens": [{
+        "token_id": 123,
+        "strategy": 1
+      },{
+        "token_id": 124,
+        "strategy": 1
+      }]
+    },
+    "memo": ""
+  }
+]' -p alice
+```
+:::
+
+## Using purchase option for exchange
+
+Exchanging a uniq is similar to swapping it but this time instead of user loosing access to his Uniq and burning a Uniq it will simply be transferred to a dedicated account. This may be useful in case Uniqs have valuable metadata attached to them and you will later utilize those Uniqs in some other scenario. Example below configures the receiver of transferred Uniqs as `1aa2aa3aa4aa` account and it also must be a Uniq from factory 45 to be able to use this purchase option. Specifying `transfer_tokens_receiver_account` is mandatory in such scenario. Note how `strategy` is set to 2
```

Comment:
```suggestion
Exchanging a Uniq is similar to swapping it but this time instead of a user losing access to his Uniq and burning a Uniq it will simply be transferred to a dedicated account. This may be useful in case Uniqs have valuable metadata attached to them, and you will later utilize those Uniqs in some other scenario. The example below configures the receiver of transferred Uniqs as `1aa2aa3aa4aa` account, and it also must be a Uniq from factory 45 to be able to use this purchase option. Specifying `transfer_tokens_receiver_account` is mandatory in such scenario. Note how `strategy` is set to 2.
```